cgi-bin/DW/Media.pm

```diff
@@ -11,7 +11,7 @@
 # Authors:
 #      Mark Smith <mark@dreamwidth.org>
```

Comment:
You can add your name to the authors list when you're editing files, if you want. Up to you!

Reply:
http://wiki.dreamwidth.net/wiki/index.php/File_headers#Authors_list says it's too minor a change for me to. Are you proposing an official policy change?

Reply:
Those are guidelines. If it was getting out of hand I'd push back, I just figured I'd mention it since I hadn't seen you do that in a while so I wasn't sure if you were considering it or not.